shacl12-vocabularies/shacl-shacl.ttl

```diff
@@ -161,17 +163,15 @@ shsh:ShapeShape
 	] ;
 	sh:property [
 		sh:path sh:in ;
-		sh:maxCount 1 ;                 # in-maxCount		
-		sh:node shsh:ListShape ;        # in-node
+		sh:maxCount 1 ;                     # in-maxCount
+		sh:minListLength 1 ;                # in-minListLength
+		sh:memberShape [ a sh:NodeShape ] ; # in-memberShape
```

Comment:
```suggestion
```

Removing as `minListLength` can now be declared without needing to have `sh:memberShape` present